src/Bicep.Cli/Commands/RootCommand.cs

```diff
@@ -74,28 +75,31 @@ bicep build file.bicep --outdir dir1
       bicep build file.bicep --outfile file.json
       bicep build file.bicep --no-restore
       bicep build file.bicep --diagnostics-format sarif
+      bicep build --pattern './dir/*.bicep'
```

Comment:
I thinking --patern './dir/**/*.bicep' would be a better example for these, some will not be aware of this possibility.